src/docs/product/integrations/integration-platform/internal-integration.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Internal Integrations
+sidebar_order: 2
+description: "Learn more about Sentry's internal integrations."
+---
+
+Internal integrations are meant for custom integrations unique to your organization. They can also be as simple as an organization-wide token. Whether you are using just the API or all the Integration Platform features combined, internal integrations are for use within a single Sentry organization.
+
+Internal integrations don't require an OAuth flow. You receive an org-wide Auth Token immediately after creation.
+
+![Diagram showing process difference between public and internal integrations.](authentication-flow.png)
+
+For an example of how to build an internal integration, see [our Round Robin Issue Assignment integration](https://blog.sentry.io/2019/11/21/customize-your-sentry-workflow-a-sample-internal-integration) (or jump straight to [the code on GitHub](https://github.com/getsentry/sentry-round-robin)).
+
+## Installation
+
+Creating an internal integration will automatically install it on your organization.
+
+## Auth Tokens
```

Comment:
As per my other comment on the Integration Platform page, I think we can reduce the duplicate stuff by changing this to something like:

> ## Authentication Tokens
> Internal integrations automatically generate an [Authentication Token](/integration-platform#authentication-tokens) when configured. Should you need multiple tokens, or you need to get a new one, you can go to **Settings > Developer Settings > [Your Internal Integration]** and do so.
> You can have up to 20 tokens at a time for each internal integration.
> These tokens do not expire automatically, but you can manually revoke them as needed.
> [image]

This would also remove the a lot of the repeated text from this page and the Public Integrations page.